package.json

```diff
@@ -18,13 +13,23 @@
   "eslintConfig": {
     "extends": "react-app"
   },
+  "homepage": "https://meganandringo.com",
   "browserslist": [
     ">0.2%",
     "not dead",
     "not ie <= 11",
     "not op_mini all"
   ],
+  "dependencies": {
+    "emotion": "^10.0.9",
+    "reach-router": "0.0.0",
```

Comment:
This installation of Reach Router is incorrect. Checking `node_modules`, the contents of this package consist of a single `package.json` file (no JavaScript files found).

![](https://www.dropbox.com/s/pkm52s1upes5fee/Screenshot%202019-04-25%2022.20.20.png?raw=1)

Execute the following command to install the Reach Router:

```bash
$ npm install --save @reach/router
```

Reply:
Documentation: https://reach.tech/router